packages/core/src/httpClient/httpHeaders.ts

```diff
@@ -0,0 +1,19 @@
+export interface HttpHeaders {
+  [key: string]: string;
+}
+
+/** @hidden */
+export function getHeaderField(
+  headers: HttpHeaders,
+  key: string
+): string | undefined {
+  const lowercaseKey = key.toLowerCase();
+
+  for (const headerKey in headers) {
```

Comment:
Is there a function find such that you can write ```headers.find(headerKey => headerKey.toLowerCase() == lowercaseKey)``` or something like that?

Reply:
Not that I know of. You can do some casting and then map it, but it becomes fairly verbose imo.